packages/app/src/systems/Home/pages/components/HomeActions/HomeActions.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Flex } from "@fuel-ui/react";
+
+export type HomeActionsProps = {
+  isDisabled?: boolean;
+};
+
+export const HomeActions = ({ isDisabled }: HomeActionsProps) => {
```

Comment:
Idk, but I think this is kinda an "over abstraction", these buttons will remain in home always and don't have a specific case to be tested and split into a component